r/vignettes/developing.Rmd

```diff
@@ -60,36 +71,38 @@ brew install apache-arrow
 brew install apache-arrow --HEAD
 ```
 
+### Windows and Linux
+
 On Windows and Linux, you can download a .zip file with the arrow dependencies from the
 nightly repository.
-Windows users then can set the `RWINLIB_LOCAL` environment variable to point to that
-zip file before installing the `arrow` R package. On Linux, you'll need to create a `libarrow` directory inside the R package directory and unzip that file into it. Version numbers in that
-repository correspond to dates, and you will likely want the most recent.
 
 To see what nightlies are available, you can use Arrow's (or any other S3 client's) S3 listing functionality to see what is in the bucket `s3://arrow-r-nightly/libarrow/bin`:
 
 ```
 nightly <- s3_bucket("arrow-r-nightly")
 nightly$ls("libarrow/bin")
 ```
+Version numbers in that repository correspond to dates.
 
-## Developer environment setup
+#### Windows
 
-If you need to alter both the Arrow C++ library and the R package code, or if you can’t get a binary version of the latest C++ library elsewhere, you’ll need to build it from source too. This section discusses how to set up a C++ build configured to work with the R package. For more general resources, see the [Arrow C++ developer
-guide](https://arrow.apache.org/docs/developers/cpp/building.html).
+Windows users then can set the `RWINLIB_LOCAL` environment variable to point to the zip file containing the arrow dependencies before installing the arrow R package. 
+
+#### Linux
+
+On Linux, you'll need to create a `libarrow` directory inside the R package directory and unzip the zip file containing the arrow dependencies into it. 
```

Comment:
```suggestion
On Linux, you'll need to create a `libarrow` directory inside the R package directory and unzip the zip file containing the compiled arrow binary files into it. 
```

Reply:
It does include some dependencies (and technically libarrow *is* a dependency of the R pacakge), but I think this is a clearer description of what we are doing here.

Reply:
Thanks!